scripts/verify-phase-57-r-supportability-pre-refactor-boundary.sh

```
@@ -0,0 +1,186 @@
+
+required_phrases=(
+  "- **Status**: Accepted for Phase 57.R.1 planning and review."
+  "- **Date**: 2026-05-04"
+  "- **Related Issues**: #1224, #1225, #1226, #1227, #1228, #1229"
+  "- **Depends On**: #1215"
+  "This ADR is documentation and verification only. It does not move implementation code, change runtime behavior, change public service facade behavior, change CLI/API behavior, change restore validation semantics, change admin behavior, change RBAC behavior, change persistence, change schema, change approval, change execution, change reconciliation, change AI behavior, or implement Phase 58 supportability features."
+  "AegisOps control-plane records remain authoritative for alert, case, evidence, recommendation, action review, approval, action request, delegation, execution receipt, reconciliation, audit event truth, limitation, release, gate, restore validation, and closeout truth."
+  "This ADR, extracted helpers, UI posture data, supportability projections, test fixtures, verifier output, issue-lint output, browser state, local cache, admin configuration, role matrix documents, and operator-facing summaries remain subordinate planning, implementation, policy, or validation evidence."
+  "If provenance, scope, auth context, record linkage, snapshot consistency, or restore-validation signals are missing, malformed, ambiguous, or only partially trusted, Phase 57.R work must fail closed and preserve the current guard instead of inferring success."
+  "That path is not present in this checkout, so this repo-owned ADR uses the tracked Phase 55-57 closeout handoff text and the tracked persona/gate/supportability docs as the local verification source."
+  '`apps/operator-ui/src/app/operatorConsolePages/adminPages.tsx`'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)